src/electron/electron/utils.cljs

```diff
@@ -37,7 +37,7 @@
 
 (defn fix-win-path!
   [path]
-  (when path
+  (when (not-empty path)
```

Comment:
can just use `seq`: (when (seq path) ...)

Reply:
I prefer to use them interchangeably. Since `non-empty` is more semantic meaningful.